### DISCUSSION TITLE
What Can I Do with my Terra AC?

### DISCUSSION THREAD

**Discussion by ilmisu** [lbbrhzn/ocpp#960] [Q&A]
Hi all,
I've installed OCPP integration onto my HA instance.
I've been able to link the Wallbox to the OCPP Server.
![image](https://github.com/lbbrhzn/ocpp/assets/50550633/c47e7c84-b5db-4f2c-a322-f0f61f5b5af0)
but, if I play a bit with these controls, nothing happens. they come back to the initial status, If I switch it off, it becomes on itself, and vice versa.
I'd like to manage the charging accoridng to some automation, but I have some doubts at the moment in being able to do it.
any hint or any example?
cheers.

**Comment by lbbrhzn** [comment]
The functionality available through the integration depends in the feature profiles supported by the charger. Based on this [document](https://library.e.abb.com/public/8f07987a3a284da6bf4e4f8f53cd6502/ABB_Terra_AC_Charger_OCPP1.6_ImplementationOverview%20_v1.8_FW1.6.6.pdf?x-sign=tTcutyEN9wSOot47Re+elEnZ3aWLPj0tmYsDHirusunfbt8F+1yPoNwWiI9lm04T) it supports the SmartCharging profile, so you should be able to set the charging current dynamically.

**Comment by ilmisu** [comment]
ok, but the switch "availability" and "charge control" are made for what?



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
31/10/23,
16:14:02

Il giorno mar 31 ott 2023 alle ore 13:08 lbbrhzn ***@***.***>
ha scritto:

**Comment by stefanschaedeli** [comment]
where your able to clear this ? I'm considering as well the ABB Terra but need the dynamic load setting (set the charge amps during the charing). Do you know if this is possible?
